docs/content/en/_index.md

```diff
@@ -18,19 +18,17 @@ Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandr
 
 ## Features for single- and multi-cluster Kubernetes environments
 
-| K8ssandra v1.4.x      | K8ssandra Operator v1.0.0 |
+| K8ssandra       | K8ssandra Operator for enhanced capabilities |
 | ----------- | ----------- |
-| K8ssandra v1.4.x is our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
-| Start in the K8ssandra v1.4.x [install](https://docs-staging-v1.k8ssandra.io/install/local/) topics, which include the steps for single-cluster installs on local or cloud-provider Kubernetes platforms. | Start in the K8ssandra Operator v1.0.0 [local install]({{< relref "install/local" >}}) topic, which focuses on multi-cluster or single-cluster installs on local dev  **kind** Kubernetes, using helper scripts, `helm`, or `kustomize`.
+| K8ssandra v1.x was our initial implementation. It provides a set of separate Helm charts you can use to configure and deploy Apache Cassandra&reg; into a single-cluster, single-region Kubernetes environment. | K8ssandra Operator v1.0.0 is our most recent offering. It combines API, management, &amp; observability features under the control of a unified operator. Important enhancements include **multi-cluster** and **multi-region** support for Cassandra deployments in Kubernetes, which enables greater scalability and availability. Single cluster/region deployments are also supported with K8ssandra Operator.|
```

Comment:
This is minor, but I think you can omit the version numbers for the operator here so that it can be more general and stand the test of time.

Reply:
Reworded...